src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -106,6 +110,7 @@ const MapOverlays = ({
                 location={building.location}
                 onPress={() => {
                   onBuildingTap(building.id);
+                  setBuildingMarkerLocation(building);
```

Comment:
same as above, might be better to fuse these two methods

Reply:
Done!